front/app/components/ProjectAndFolderCards/index.tsx

```diff
@@ -35,29 +46,35 @@ export default ({ publicationStatusFilter, ...otherProps }: BaseProps) => {
     publicationStatusFilter
   );
 
-  const publicationStatuses = useMemo(() => {
+  const publicationStatusesForCurrentTab = useMemo(() => {
     if (!currentTab) return;
 
     return currentTab === 'all' ? publicationStatusFilter : [currentTab];
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [currentTab, publicationStatusesStringified]);
```

Comment:
React.useMemo is made for what you're doing with the stringified thing here.

Reply:
Does `useMemo` do shallow or deep comparison of what's in the dependency array? I can't find anything about this in the documentation (https://reactjs.org/docs/hooks-reference.html#usememo) or elsewhere